recipes/r-rrf/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set version = '1.9.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rrf
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RRF_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RRF/RRF_{{ version }}.tar.gz
+  sha256: 3c3c5b349861090ffd31e3187a5a7618bc2fb3faf5d55a5b7b483b769c6daa8c
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('RRF')"           # [not win]
+    - "\"%R%\" -e \"library('RRF')\""  # [win]
+
+about:
+  home: https://sites.google.com/site/houtaodeng/rrf
+
+  license: GPL-2
```

```suggestion
  license: GPL-2.0-or-later
```

derived from cran meta-data `License: GPL (>= 2)`, see also https://spdx.org/licenses/GPL-2.0-or-later.html